docs/manual/java/guide/cluster/Cluster.md

```diff
@@ -147,6 +147,20 @@ The reason for the special first seed node is to avoid forming separated islands
 
 You can read more about cluster joining in the [Akka documentation](https://doc.akka.io/docs/akka/2.6/cluster-usage.html?language=java#joining-to-seed-nodes).
 
+## Rolling Updates and Versions
```

Comment:
I don't think we should have this in the cluster documentation. This is a migration concern and should be on the migration guide.

Reply:
And only when applicable. Each time that we cut a Lagom version that's bringing a new Akka version we should verify this and update the migration guide accordingly. Or decide to not bump  Akka's version.

Reply:
I always considered the `MigrationN.md` as documentation to consider when migrating from version `N-1` to version `N`.

I added the table in `Cluster.md` to inform users in version `A.B.C` of the steps required to migrate to `A.B.latest`. I mean, If I was running `1.6.0` already I'd never think to check `Migration16.md` before migrating to `1.6.3`.

Reply:
> And only when applicable

I agree with you on that. Maybe instead of:


| from  |  to   |
| :---: | :---: |
| 1.6.0 | 1.6.2 |
| 1.6.1 | 1.6.3 |
| 1.6.2 | 1.6.3 |

We should use:

| from  |  to   |
| :---: | :---: |
| 1.6.0 | 1.6.2 |
| 1.6.1..1.6.2 | 1.6.3 |

Reply:
> I mean, If I was running 1.6.0 already I'd never think to check Migration16.md before migrating to 1.6.3

That's correct, me neither. But would you read the Cluster.md page before upgrading? 

Updating the Migration page is fragile for the reasons you mentioned, but even so is the Cluster page. Instead, we should add to the Migration page and a big callout on the release notes. We can't do more than that.

Reply:
I agree that a special mention in the release notes is necessary too. But I think we must keep an absolute track of this type of changes. See for example Akka's approach: there's both the [release notes for `2.6.5`](https://akka.io/blog/news/2020/04/30/akka-2.6.5-released) and a page with a [complete list of affected versions](https://doc.akka.io/docs/akka/current/project/rolling-update.html#2-6-5-jacksoncborserializer) in the ref docs.

Reply:
I've started a draft for the [release notes](https://github.com/lagom/lagom/releases/tag/untagged-73ecf84d318c872964e3)

Reply:
@renatocaval WDYT about the approach I suggested: release notes + absolute list in the docs (in `Cluster.md` or elsewhere)

Reply:
I agree with the idea of absolute list page.

I just really don't think this should be integrated into the `Cluster.md` page. We are mixing concerns in doing that. 

The Cluster page can have a link to it though, but we should also link to it from other sections. For instance, on the menu on the right, under "Lagom releases".